pkg/csi/service/common/topology.go

```diff
@@ -279,6 +279,16 @@ func fetchHosts(ctx context.Context, entity mo.Reference, vCenter *cnsvsphere.Vi
 				entity.Reference(), err)
 		}
 		hosts = append(hosts, hostList...)
+	case "ComputeResource":
```

Comment:
IMO we can just create an instance of `object.NewComputeResource` and call the in-built `Hosts` method on it which will give you a list of `object.HostSystem`. Convert it to  `[]*cnsvsphere.HostSystem` and return it. We do not need a separate struct for `ComputeResource` as we do not use it much in our code.

Reply:
Done